Makefile

```diff
@@ -11,7 +11,7 @@ angel:
 	@./mvnw clean test -Dversion.org.springframework.boot=1.2.8.RELEASE -Dversion.org.springframework.cloud-spring-cloud-dependencies=Angel.SR6
 
 dalston:
-	@./mvnw clean test -Dversion.org.springframework.boot=1.5.6.RELEASE -Dversion.org.springframework.cloud-spring-cloud-dependencies=Dalston.SR3
+	@./mvnw clean test -Dversion.org.springframework.boot=1.5.8.RELEASE -Dversion.org.springframework.cloud-spring-cloud-dependencies=Dalston.SR4
```

Comment:
I think we can remove this target it's basically what default one does. You could create one for Camden if somebody would like to run it. The travis config has to be changed too.

Reply:
Remove the whole dalston command or just the properties?

Reply:
And how should I update Travis config? Remove dalston step?

Reply:
Makefile: remove dalston target, we don't need it is default now, in travis remove https://github.com/opentracing-contrib/java-spring-cloud/blob/master/.travis.yml#L27-L28 and in dalston do make default